### PR TITLE
Kapiljain

### PR DESCRIPTION
**Pull request by kapiljain786**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/test/resources/application.conf [hunk 1]
```diff
@@ -89,25 +88,73 @@ cnp_sanbox {
   createCaseUrl = "caseworkers/6687/jurisdictions/PROBATE/case-types/GrantOfRepresentation/cases"
 }
 
-cnp_aat {
-  jurisdiction = "sscs" //TODO get the jurisdiction from the url
+cnp_sprod {
+  jurisdiction = "autotest1"
```
Comment (mario-paniccia) [inline src/test/resources/application.conf]: I'm quite sure this parameter is no longer needed. Can you please remove it and verify that ev still work?

File: src/test/resources/application.conf [hunk 2]
```diff
@@ -89,25 +88,73 @@ cnp_sanbox {
...
+  getCaseUrl = "caseworkers/176475/jurisdictions/AUTOTEST1/case-types/AAT/cases/1525778314235511"
+  createCaseUrl = "caseworkers/176475/jurisdictions/AUTOTEST1/case-types/AAT/cases"
+  getUserProfileUrl = "caseworkers/176475/profile"
+  searchCases = "caseworkers/176475/jurisdictions/AUTOTEST1/case-types/AAT/cases?state=CaseCreated"
+  getPaginationMetadataUrl = "caseworkers/176475/jurisdictions/AUTOTEST1/case-types/AAT/cases/pagination_metadata?state=CaseCreated"
+  saveEventUrl = "caseworkers/176475/jurisdictions/AUTOTEST1/case-types/AAT/cases/1525778314235511/events"
+}
+
+cnp_aat {
+  jurisdiction = "divorce" //TODO get the jurisdiction from the url
```
Comment (mario-paniccia) [inline src/test/resources/application.conf]: same as above

File: src/test/scala/uk/gov/hmcts/ccd/util/PerformanceTestsConfig.scala [hunk 3]
```diff
@@ -17,6 +17,16 @@ trait PerformanceTestsConfig {
   val CaseDefinitionUrl = config.getString("caseDefinitionUrl")
   val MaxSimulationDuration = config.getInt("maxSimulationDurationMinutes")
   val RequestRateSec = Option(System.getProperty("request.rate")).map(_.toDouble)
+  val minThinkTime = config.getInt("minthinktime")
```
Comment (mario-paniccia) [inline src/test/scala/uk/gov/hmcts/ccd/util/PerformanceTestsConfig.scala]: in scala constants starts with uppercase. can you change all of the new constants to have first letter uppercase please?

File: src/test/resources/application.conf (hunk 1 above)
Comment (kapiljain786) [reply src/test/resources/application.conf]: Removed Jurisdiction parameter and tested OK locally in sprod environment

File: src/test/scala/uk/gov/hmcts/ccd/util/PerformanceTestsConfig.scala (hunk 3 above)
Comment (kapiljain786) [reply src/test/scala/uk/gov/hmcts/ccd/util/PerformanceTestsConfig.scala]: Done

**Pull request merged**